dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/task/etl/EtlParameters.java

```diff
@@ -0,0 +1,191 @@
+package org.apache.dolphinscheduler.common.task.etl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.dolphinscheduler.common.task.AbstractParameters;
+
+/**
+ * Sql/Hql parameter
+ */
+public class EtlParameters extends AbstractParameters {
+
+    /**
+     * data source type，eg  MYSQL, POSTGRES ...
+     */
+    private String dsType;
+
+    /**
+     * datasource id
+     */
+    private int datasource;
+
+    /**
+     * data target type，eg  MYSQL, POSTGRES ...
+     */
+    private String dtType;
+
+    /**
+     * datatarget id
+     */
+    private int datatarget;
+
+    /**
+     * sql
+     */
+    private String sql;
+
+    /**
+     * target table
+     */
+    private String targetTable;
+
+    /**
+     * Pre Statements
+     */
+    private List<String> preStatements;
+
+    /**
+     * Post Statements
+     */
+    private List<String> postStatements;
+
+    /**
+     * speed byte num
+     */
+    private int jobSpeedByte;
+
+    /**
+     * speed record count
+     */
+    private int jobSpeedRecord;
+
+    public String getDsType() {
+        return dsType;
+    }
+
+    public void setDsType(String dsType) {
+        this.dsType = dsType;
+    }
+
+    public int getDatasource() {
+        return datasource;
+    }
+
+    public void setDatasource(int datasource) {
+        this.datasource = datasource;
+    }
+
+    public String getDtType() {
+        return dtType;
+    }
+
+    public void setDtType(String dtType) {
+        this.dtType = dtType;
+    }
+
+    public int getDatatarget() {
+        return datatarget;
+    }
+
+    public void setDatatarget(int datatarget) {
+        this.datatarget = datatarget;
+    }
+
+    public String getSql() {
+        return sql;
+    }
+
+    public void setSql(String sql) {
+        this.sql = sql;
+    }
+
+    public String getTargetTable() {
+        return targetTable;
+    }
+
+    public void setTargetTable(String targetTable) {
+        this.targetTable = targetTable;
+    }
+
+    public List<String> getPreStatements() {
+        return preStatements;
+    }
+
+    public void setPreStatements(List<String> preStatements) {
+        this.preStatements = preStatements;
+    }
+
+    public List<String> getPostStatements() {
+        return postStatements;
+    }
+
+    public void setPostStatements(List<String> postStatements) {
+        this.postStatements = postStatements;
+    }
+
+    public int getJobSpeedByte() {
+        return jobSpeedByte;
+    }
+
+    public void setJobSpeedByte(int jobSpeedByte) {
+        this.jobSpeedByte = jobSpeedByte;
+    }
+
+    public int getJobSpeedRecord() {
+        return jobSpeedRecord;
+    }
+
+    public void setJobSpeedRecord(int jobSpeedRecord) {
+        this.jobSpeedRecord = jobSpeedRecord;
+    }
+
+    @Override
+    public boolean checkParameters() {
+        if (!(datasource != 0 && StringUtils.isNotEmpty(dsType) && StringUtils.isNotEmpty(sql))) {
```

Comment:
Whether these two if condition are considered to be combined into one？

Reply:
thx, let me fix it